### PR TITLE
Update README and fix regex bug

### PR DESCRIPTION
**Pull request by narthur157**
Forgot about this bug I fixed locally a while ago, adding it to this commit and responded to Joel's comments on the patch PR here.

old PR: https://github.com/behance/ractive-partials/pull/5

### CODE REVIEW

File: src/ractive-partials.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import text from './text';
 import module from 'module';
 
 // Finds all '{{> partialName }}' in the template
-let findPartial = /{{>\s?([^\s]+)\s?}}/gi;
+let findPartial = /{{>\s?([^\s]+?)\s?}}/gi;
```
Comment (mrjoelkemp) [inline src/ractive-partials.js]: What's an example of why this is necessary?

Comment (mrjoelkemp) [inline src/ractive-partials.js]: The change, that is

Comment (mrjoelkemp) [inline src/ractive-partials.js]: Oh this is a bug fix. We typically link to the actual issue in the commit message or PR body.

Comment (narthur157) [inline src/ractive-partials.js]: Yeah, pretty obscure, but this is why https://regex101.com/r/qP6qJ9/4

Comment (narthur157) [inline src/ractive-partials.js]: Explanation here https://medium.com/@narthur157/github-com-behance-ractive-partials-868467ab17c1

**Comment by Aintaer** [comment]
:cool:

**Pull request merged**